packages/cloudflare/src/cli/commands/populate-cache.ts

```diff
@@ -100,6 +101,25 @@ export async function populateCache(
         logger.info(`Successfully populated cache with ${assets.length} assets`);
         break;
       }
+      case KV_CACHE_NAME: {
+        logger.info("\nPopulating KV incremental cache...");
+
+        const assets = getCacheAssetPaths(options);
+        for (const { fsPath, destPath } of tqdm(assets)) {
+          runWrangler(
+            options,
+            [
+              "kv key put",
+              JSON.stringify(destPath),
+              "--binding NEXT_INC_CACHE_KV",
```

Comment:
- `NEXT_INC_CACHE_KV` should be shared between the handler and here (see #538)
- we probably want to check if `NEXT_INC_CACHE_KV` exists instead of erroring as many times as there are assets